conductor_api/src/conductor/base.rs

```diff
@@ -1474,6 +1477,40 @@ pub mod tests {
         assert_eq!(result, JsonString::from(RawString::from("Holo World")));
     }
 
+    #[test]
+    fn basic_bridge_call_error() {
+        let config = load_configuration::<Configuration>(&test_toml(10041, 10042)).unwrap();
+        let mut conductor = Conductor::from_config(config.clone());
+        conductor.dna_loader = test_dna_loader();
+        conductor.key_loader = test_key_loader();
+        conductor
+            .boot_from_config()
+            .expect("Test config must be sane");
+        conductor
+            .start_all_instances()
+            .expect("Instances must be spawnable");
+        let caller_instance = conductor.instances["bridge-caller"].clone();
+        let mut instance = caller_instance.write().unwrap();
+
+        let cap_call = {
+            let context = instance.context();
+            make_cap_request_for_call(
+                context.clone(),
+                Address::from(context.clone().agent_id.address()),
+                "call_bridge_error",
+                JsonString::empty_object(),
+            )
+        };
+        let result = instance.call("test_zome", cap_call, "call_bridge_error", "{}");
+
+        assert!(result.is_ok());
+        assert_eq!(
+            result.unwrap(),
+            JsonString::from("{\"Err\":{\"Internal\":\"{\\\"kind\\\":{\\\"ErrorGeneric\\\":\\\"{\\\\\\\"code\\\\\\\":-32602,\\\\\\\"message\\\\\\\":\\\\\\\"Holochain Instance Error: Zome function \'non-existent-function\' not found in Zome \'greeter\'\\\\\\\"}\\\"},\\\"file\\\":\\\"core/src/nucleus/ribosome/runtime.rs\\\",\\\"line\\\":\\\"225\\\"}\"}}"
+            )
```

Comment:
So this test does not test the log output but I tested the log output by running this test with `--nocapture` and watching the output.
We are not doing log testing (yet?) and I did not add log tests (as in testing for the existence of specific log outputs) because it feels like overkill.